src/main/java/de/tum/in/www1/artemis/domain/ProgrammingAssessmentUpdate.java

```diff
@@ -0,0 +1,39 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.time.ZonedDateTime;
+
+/**
+ * A helper object that aggregates feedback list, complaint response, score, result string and completion date which are used to update an assessment after a complaint.
+ */
+public class ProgrammingAssessmentUpdate extends AssessmentUpdate {
```

Comment:
Why do we need this?

Reply:
This is similar to how we do it with other types of exercises, but in case of programming exercise tutor can also change result string, completion date and score. But again I am not sure if we should allow to change manual result infinitely as we have it now